hack/env.sh

```diff
@@ -32,3 +32,4 @@ export RELEASE_VERSION=4.7.0
 export OPERATOR_NAME=sriov-network-operator
 export RESOURCE_PREFIX=openshift.io
 export ENABLE_ADMISSION_CONTROLLER=${ENABLE_ADMISSION_CONTROLLER:-"true"}
+export CLUSTER_TYPE=${CLUSTER_TYPE:-openshift}
```

Comment:
This will only be used by `make run` which is only for development usage. If developers want to run in vanilla k8s, they can run `CLUSTER_TYPE=kubernetes make run`. In the deploy case, we already have this environment variable in the openshift images.